src/translators/base_translator.py

```diff
@@ -17,7 +17,7 @@ def __init__(self, host, port, db_name):
         self.port = port
         self.db_name = db_name
 
-    def __enter__(self):
```

Comment:
i don't think you need to change this method, the idea should be to replace the connection manager with sqlalchemy "create engine", but without changing interfaces